### PR TITLE
 Added config

### PR DESCRIPTION
**Pull request by esikachev**
We need to use config-file. It's temporary resolution

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -7,3 +7,35 @@ My-dev
 **My-dev information**
 
 This is a utility for developers.
+
+
+**Installation**
+
+1. Clone project
+
+2. Install project. You have a 2 ways:
```
Comment (zemuvier) [inline README.rst]: You have 2 ways

File: README.rst [hunk 2]
```diff
@@ -7,3 +7,35 @@ My-dev
...
+..
+
+2.2 Install in virtualenv    
+
+.. sourcecode:: console
+
+    $ tox -e venv --notest
+..
+
+3. Create a config-file in home-directory on in project-directory ``.my.conf``
```
Comment (zemuvier) [inline README.rst]: Create a config-file in home-directory or (?) in project-directory ``.my.conf``

File: README.rst [hunk 3]
```diff
@@ -7,3 +7,35 @@ My-dev
...
+
+2.2 Install in virtualenv    
+
+.. sourcecode:: console
+
+    $ tox -e venv --notest
+..
+
+3. Create a config-file in home-directory or in project-directory ``.my.conf``
+   with host and username. Example of config-file you can found in ``etc/my_dev/.my.conf.sample``
```
Comment (zemuvier) [inline README.rst]: Don't forget about dot in the end of each line

**Comment by esikachev** [comment]
@zemuvier ^^

**Pull request merged**